osc-rest-server/src/main/java/org/osc/core/broker/rest/server/api/VirtualizationConnectorApis.java

```diff
@@ -512,6 +510,53 @@ public Response updateVirtualSecurityPolicyBindings(@Context HttpHeaders headers
         return this.apiUtil.getResponseForBaseRequest(this.bindSecurityGroupService, bindRequest);
     }
 
+    @ApiOperation(value = "Set Security Group Bindings with Service Function Chain  (Openstack Only)",
+            notes = "Adds/Update/Remove Security Group Bindings to Security Function Services.<br/>"
```

Comment:
Adds/Updates/RemoveSecurity Group Bindings to Security Function Services -> Enables adding, updating or removing service function chain bindings to security groups..

Reply:
Updated  content to Adds/Updates

Reply:
Adds/Update -> Adds/Update**s**

Reply:
corrected